src/atlas_consortia_clt/common/help.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+
+def get_help(config: Config) -> str:
+    """Returns a help message for the command line tool."""
+
+    return f"""
+        {config.name} [-h | --help] [transfer manifest-file | login | logout | whoami]
+
+        {config.consortium} Command-Line Transfer
+
+        A utility to bulk download {config.consortium} data by specifying the directories and/or
+        files in a manifest file. For detailed documentation see:
+        {config.docs_url}
+
+        Commands: One of the following commands is required:
+
+        transfer manifest-file   Transfer files specified in manifest-file (see
```

Comment:
I didn't test the `transfer` subcommand because I don't have the proper Globus setup. It's unclear to me if I should use `manifest-file` (described here) or `manifest` (specified in the code: https://github.com/x-atlas-consortia/clt/blob/266-generalize-data-manifest-tool/src/atlas_consortia_clt/common/main.py#L22), please make them consistent.